src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: $card-width;
+  background-color: $background-card-and-btn;
+  border-radius: 5px;
+  border: 1px solid $border-card;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  color: $main-color-text;
+  padding: $padding-card;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding-top: $padding-card;
+    align-self: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    color: $color-secondary;
+    padding-top: 4px;
+  }
+
+  &__stars-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 17px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__block-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 23px;
+    font-weight: 400;
+  }
+
+  &__word-price {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__btn {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
```

Comment:
do not hardcode width - better to set 100%
what if you will change the card width?